tools/c7n_azure/tests/test_hdinsight.py

```diff
@@ -38,7 +40,40 @@ def test_find_hdinsight_cluster_by_name(self):
                     'value_type': 'normalize',
                     'value': 'cctesthdinsight*'
                 }
-            ],
-        })
+            ]
+        }, validate=True)
+
         resources = p.run()
         self.assertEqual(len(resources), 1)
+
+    @patch('azure.mgmt.hdinsight.operations.ClustersOperations.resize')
+    @arm_template('hdinsight.json')
+    @cassette_name('common')
+    def test_resize_hdinsight_cluster_action(self, resize_mock):
+        p = self.load_policy({
+            'name': 'test-azure-hdinsight-resize',
+            'resource': 'azure.hdinsight',
+            'filters': [
+                {
+                    'type': 'value',
+                    'key': 'name',
+                    'op': 'glob',
+                    'value_type': 'normalize',
+                    'value': 'cctesthdinsight*'
+                }
+            ],
+            'actions': [
+                {
+                    'type': 'resize',
+                    'count': 1
+                }
+            ]
+        }, validate=True)
```

Comment:
Winston, sorry for the confusion earlier. Can you please remove validate piece here & maybe add a validation in `test_hdinsight_schema_validate`? 
We don't want to have explicit validation in actionable tests due to it being an expensive operation.. I think we have a solution, but it is not compatible with Py2.7 so we will flip `validate` to be True by default later.

(The reason I'm asking to remove it from here is people tend to copy-paste some of the test code, so we don't want it to spread)